docs/userguide/harness/user/reference.rst

```diff
@@ -26,12 +26,12 @@ All arguments are optionals.
     ``--trigger-uids``, "Specifies the list of triggers uids to run (list or logic expression)"
     ``--trigger-groups``, "Specifies the list of triggers groups to run (logic expression)"
     ``--devices``, "List of devices to connect to (in case of no mapping datafile)"
-    ``--subsection-datafile``, "Full path/name to the common_(setup/cleanup) subsection datafile"
-    ``--mapping-datafile``, "Full path/name to the mapping datafile"
-    ``--verification-datafile``, "Full path/name to the verification datafile"
-    ``--trigger-datafile``, "Full path/name to the trigger datafile"
-    ``--config-datafile``, "Full path/name to the config datafile"
-    ``--pts-datafile``, "Full path/name to the PTS datafile"
+    ``--subsection-datafile``, "Full path/name or URL to the common_(setup/cleanup) subsection datafile"
```

Comment:
just want to confirm. pyATS command's help is also updated?

Reply:
yes, https://github.com/CiscoTestAutomation/pyats/pull/119/files